9.1.Индексация/9.1.8.py

```diff
@@ -0,0 +1,4 @@
+st = input()
+
+for i in range(-1, -len(st) - 1, -1):
+    print(st[i])
```

Comment:
Чутка усложнил, можно попороще:
```suggestion
for i in range(1, len(st) + 1):
    print(st[-i])
```